src/Microsoft.DotNet.Wpf/src/PresentationCore/ModuleInitializer.cs

```diff
@@ -14,11 +25,16 @@ internal static class ModuleInitializer
     /// as the module constructor for DirectWriteForwarder would do this anyway.
     /// </summary>
 #pragma warning disable CA2255
+#if NET
     [ModuleInitializer]
     public static void Initialize()
     {
         IsProcessDpiAware();
 
+#else
```

Comment:
I would expect if we defined a "ModuleInitializer" attribute someone where for the .NET Framework build, C# would do the right thing here and turn this into a module initializer, did you try that?

Reply:
Yeah - that works - it's an OOBable feature https://sharplab.io/#v2:EYLgHgbALANALiATgVwHYwCYgNQB8D0+ABAMoCmZRAFnHAA4DOIhAhgNYsB0Atg/qmQDuAWgDGAe1QNxAGzLC4ZbnRktFRAGbjERbtsoBLVFsTc1ByQFgAUAAEADEVsBGTgCU0cA9zKcAwuLKBnKI5IgAbgaiZAwA3Da2AEzONgnOAJwAFABEABJkMjLiMEQA6toyGACE2QCU8dYJAMxOzhBOiUQAsuIYyHIAkqgGXiwyBgBeZIg2AN42RItEANo9fYPDo+NTiAC6C0u2LS7ttlBEQyMGY5NkmbUHi/PWS6+tWdlr/Yab19vTRFEYzkGE4dQarwAvjZoY04URUCwfAw6Cxoq13J5vL4AkEQmFItEGAtnq9lgBBWiIAzAZCKACqDBYAHM7pS4NTaYoACosRCsuAMThdMhwKi9ErkwriQRdfpeFSUAC8mjGDDItX2L0OLXVYzIGA63V630uW1uiHZnLplBARCtNJtRFmRFhrqAA===